doc/kubernetes/modules/ROOT/partials/running/infinispan-credentials.adoc

```diff
@@ -0,0 +1,37 @@
+. Configure the credential to access the Infinispan cluster.
++
+Keycloak needs this credential to be able to access the cluster.
```

Comment:
"Keycloak needs this credential to be able to authenticate with the {ispn} cluster."?